charts/piraeus/templates/external-metrics-service.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if (.Values.externalController).url }}
+---
+{{- $url := urlParse .Values.externalController.url -}}
+{{- $host := regexSplit ":" (get $url "host") -1 -}}
+{{- $ipAddr := index $host 0 -}}
+{{- $port := index $host 1 }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ include "piraeus-operator.fullname" . }}-external-controller-metrics-service
+  labels:
+  {{- include "piraeus-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: linstor-external-controller
+subsets:
+  - addresses:
+    - ip: {{ $ipAddr }}
+    ports:
+    - name: metrics
+      port: {{ $port }}
+      protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "piraeus-operator.fullname" . }}-external-controller-metrics-service
+  labels:
+  {{- include "piraeus-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: linstor-external-controller
+spec:
+  type: ExternalName
+  externalName: {{ $ipAddr }}
+  clusterIP: ""
```

Comment:
According to https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors, you don't need to set `type`, `externalName` or `clusterIp`.

Reply:
Fixed